api/container_registries_gcp_gar.go

```diff
@@ -52,13 +52,14 @@ func (reg GcpGarIntegration) ContainerRegistryType() containerRegistryType {
 }
 
 type GcpGarData struct {
-	Credentials    GcpCredentialsV2    `json:"credentials"`
-	RegistryDomain string              `json:"registryDomain"`
-	RegistryType   string              `json:"registryType"` // always "GCP_GAR"
-	LimitByTag     []string            `json:"limitByTag"`
-	LimitByLabel   []map[string]string `json:"limitByLabel"`
-	LimitByRep     []string            `json:"limitByRep"`
-	LimitNumImg    int                 `json:"limitNumImg"`
+	Credentials      GcpCredentialsV2    `json:"credentials"`
+	RegistryDomain   string              `json:"registryDomain"`
+	RegistryType     string              `json:"registryType"` // always "GCP_GAR"
+	LimitByTag       []string            `json:"limitByTag"`
+	LimitByLabel     []map[string]string `json:"limitByLabel"`
+	LimitByRep       []string            `json:"limitByRep"`
+	LimitNumImg      int                 `json:"limitNumImg"`
+	NonOSPackageEval bool                `json:"nonOsPackageEval"`
```

Comment:
nit: for consistency let's make the json say `nonOSPackageEval` to match the field name.

Reply:
We discussed this and we can't change it since our APIs return this exact field.